test/functorch/test_ops.py

```diff
@@ -368,6 +368,10 @@ def is_inplace(op, variant):
     # 'tensor_split' not composite compliant, see vjp_fail
 }
 
+skip_noncontig = {
+    '_batch_norm_with_update',
```

Comment:
hmm @albanD do we have a better way for an OpInfo to mark itself as an xfail for noncontiguous tests? (or maybe no, because there isn't a dedicated "non-contiguous" test for OpInfos, and it just depends on the particular sample inputs).

Also @andrewor14 - why do we not support the noncontiguous input case? (and if we can't fix it, can you add a comment explain why). I vaguely remember from our conversation that cudnn didn't support non-contiguous inputs, but our backend selection logic is supposed to not choose cudnn in that case. But maybe there's a different reason I'm not remembering properly.

Reply:
It's because we decided to move the `arg.contiguous()` calls to the parent, which in this case will be `aten.batch_norm`. We need to do that because we want the backward op to get the exact same (contiguous) input as the forward op, so we don't have to call `input.contiguous()` twice. Since the parent is already forcing the args to be contiguous, doing it again in `aten._batch_norm_with_update` will be unnecessary and potentially costly.

As for the OpInfo comment, I agree, I think we should eventually expose something like `supports_noncontiguous_args=False` so downstream tests can read this and skip the noncontiguous checks.